cmd/bosun/sched/check.go

```diff
@@ -482,16 +482,20 @@ var bosunStartupTime = utcNow()
 
 func (s *Schedule) findUnknownAlerts(now time.Time, alert string) []models.AlertKey {
 	keys := []models.AlertKey{}
-	if utcNow().Sub(bosunStartupTime) < s.Conf.CheckFrequency {
+	if utcNow().Sub(bosunStartupTime) < s.SystemConf.GetCheckFrequency() {
 		return keys
 	}
 	if !s.AlertSuccessful(alert) {
 		return keys
 	}
-	a := s.Conf.Alerts[alert]
+	a := s.RuleConf.GetAlert(alert)
 	t := a.Unknown
 	if t == 0 {
-		t = s.Conf.CheckFrequency * 2 * time.Duration(a.RunEvery)
+		runEvery := s.SystemConf.GetDefaultRunEvery()
```

Comment:
I think we should go back to precalculating runevery per alert. This also looks to me like a reason default run every and check freq should be moved to rule conf again.